src/lib/web-component/log-view/log-view.css

```diff
@@ -20,6 +20,7 @@
 
 .log-details {
   margin: 50px 15px 10px;
+  white-space: pre-wrap;
```

Comment:
Given that you are using the CSS to make the element to be treat the whitespaces as a pre element, then you can leave the `.log-details` element as a `div` (in the end this is just a temporary workaround to make the log details a bit more readable in the short term, to be just used until you will be able to spent some time on making the sidebar details more complete and useful).

To make the log details a bit more readable, along with this temporary styling change you should also change the related `JSON.stringify` call to make it "pretty print" (indent it basically) a bit the JSON data in the generated string (If you have never used the other `JSON.stringify` parameters before you can take at this doc page: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify).

I guess that there may be some other css tweaks to make the content fit well into the page layout, e.g. you could probably make the wrapper element to show a scrollbar if the content is overflowing the wrapper element, but don't spend too much more time than that in this PR, this is just a temporary measure (and the activity log page in general could likely use some more work on its CSS to improve its layout, but it can definitely be deferred to a separate PR).

Reply:
done. Thanks :-).